test/utils/helper.js

```diff
@@ -51,6 +51,12 @@ function TestVector () {
 }
 
 TestVector.wrongRealmPublicKey = 'MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAikGBvYGniAJ59ZjpaSDw2o+j40Ila/dWfN8qA1dzXJesH9Z1sZrcevJB+rfxoZDaWMz2l9Q3OxG/qolTpsQl8NBdb5tymic9qDkAIsiyKThzjcfs5lOSxfnkHn6+Z0QbrYnXQs/cGvQ1Ai81M1M1O6BHDWu05n8c977h+BsfLmqGj7MZZj9gw9RM84RIKDGHTbFh9YyXBJVtqbOhRD7hcB0O9olDZb7mQ5A8gsMctcUhsVBy3xKCLMD41XU92rQ9FAlsV9mBglLqaVWr2mxQItN3lgjE02L8UyorI3T0uprIsnv7B2NwUC5ZhwZGfnBznUPVrT6makEJklpg5if3qQIDAQAB';
+TestVector.logoutValidPayload = 'eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6IkZKODZHY0YzalRiTkxPY280TnZaa1VDSVVtZllDcW9xdE9RZU1mYmhObEUifQ.eyJpYXQiOjE1NTY2MTcwNzksInJlc291cmNlIjoiYWRtaW5hcHAzIiwiYWN0aW9uIjoiTE9HT1VUIiwibm90QmVmb3JlIjoxNTg3MDQ3NTM3fQ.NplTHo8JuwtmUbpp3AHjM3c6rn7g_xGWegC-b8Gg7V2QoN9vPRb9oCc9fdD7qWKpXLgfNtvtTIJnIIP5O_ux7Jt_SQyNtwoPmf5k_EFmm7JSxPnVfVA36BJbGDJu_BiNbktGgpNVZR5HnAkawhsLLo05S0edFfnbs4N9a_4W_YM';
```

Comment:
If I understand this correctly, a test using this logoutValidPayload will start to fail on 16th April next year because of the notBefore policy set to value: 1587047537? Shouldn't we create tokens in some other way? For example, setting notBefore for each test run?

Reply:
@mhajas Good catch! ;) However, I believe the test will be still working (regardless `notBefore` field) - please take a look at:
https://github.com/keycloak/keycloak-nodejs-connect/blob/c6205b8f7ceca23a7a4e431f2ec829b3335e8170/test/keycloak-connect-rest-mixed-client-spec.js#L218-L222
We're just testing the logout endpoint response which will always be `200`.

When reviewing the original PR in prod, I've actually given `notBefore` in this token a lot of attention. :) And what we're doing here in the new tests is, we're not testing the logout functionality as such. We're just testing the token signature validation, regardless if the logout is actually performed. So from my PoV, the `logoutValidPayload` is ok.

Reply:
I see it now. Thank you for the explanation.